modules/ROOT/pages/kafka/kafka-connector-reference.adoc

```diff
@@ -5,252 +5,639 @@ endif::[]
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  The time unit for the operation timeout. This combines with operationTimeout to define the total default timeout for the operations that use this configuration. |  SECONDS | 
+| Zone ID a| String |  Converts the provided timestamps into `ZonedLocalDateTimes` in the results. Default value is the one provided by the system. |  | 
 |===
 
 ==== Connection Types
-[[kafka-consumer-config_basic-kafka-consumer-connection]]
-===== Kafka Basic Consumer Connection
+[[consumer-config_consumer-plaintext-connection]]
+===== Consumer Plaintext Connection
 
 ====== Parameters
 [%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Consumer Partitions a| Number |  The number of partitions to use for the consumer. |  1 | 
-| Group Id a| String |  A unique string that identifies the consumer group this consumer belongs to. |  | x
-| Bootstrap Servers a| String |  Comma-separated host-port pairs used for establishing the initial connection to the Kafka cluster. The same as the `bootstrap.servers` value you must provide to Kafka clients (producer and consumer). |  | x
-| Additional Properties a| Object |  Additional properties such as a key-value pair that you need for your connection. |  | 
+| Bootstrap Server URLs a| Array of String |  The list of servers to bootstrap the connection with the Kafka cluster. This can be a partial list of the available servers. |  | x
+| Group ID a| String |  Default group ID for all the Kafka consumers that use this configuration. |  | 
+| Consumer Amount a| Number | Determines the number of consumers the connection will initially create.  |  `1` | 
+| Maximum polling interval a| Number |  Controls the maximum amount of time the client will wait for the response of a request. If the response is not received before the timeout elapses the client resends the request; or the request fails if the specified number of retries are exhausted. This parameter can be overridden at the source level. |  `300` | 
+| Maximum Polling Interval Time Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Determines the time unit for request timeout scalar. This parameter can be overridden at source level. |  `SECONDS` | 
+| Isolation Level a| Enumeration, one of:
+
+** READ_UNCOMMITTED
+** READ_COMMITTED a| Controls how to read messages written transactionally. 
+
+If set to `read_committed`, `consumer.poll()` will return only transactional messages that have been committed. If set to `read_uncommitted`' (default), `consumer.poll()` returns all messages, even transactional messages that were aborted. Non-transactional messages are returned unconditionally in either mode. Messages are always returned in offset order. Hence, in `read_committed` mode, `consumer.poll()` returns only messages up to the last stable offset (LSO), which is the one less than the offset of the first open transaction.
```

Comment:
I don't think the wording of the last sentence makes sense - maybe it should be "... which is one less than the offset of the first open transaction."